README.md

```diff
@@ -1,8 +1,53 @@
+
+## Skills
+
+**Proficient:** Java, Linux/Bash, RDBMS/SQL, Agile/Scrum, Git
+
+**Competent:** Docker and Containerization, Intellij IDE, Visual Studio, Google Cloud Platform, REST API
+
+**Familiar:** Neural Network, Machine Learning, C#, Azure DevOps, Unit testing
+
+## Jarvis Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_NazaninAkbari](https://github.com/jarviscanada/jarvis_data_eng_NazaninAkbari)
+
+
+**Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_NazaninAkbari/tree/masterhttps://github.com/jarviscanada/jarvis_data_eng_NazaninAkbari/tree/feature/creating-DDL/linux_sql)]: Developed a Linux monitoring agent that allows a user to monitor and store hardware specifications and usage information for multiple Linux systems. The agent employs Bash Scripts to collect machine hardware information, which is then stored in a PostgreSQL database provisioned using Docker. The resource usage is fetched periodically with Crontab, and we can then manipulate the aggregated data using SQL queries.
+
+**Core Java Apps** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_NazaninAkbari/tree/masterhttps://github.com/jarviscanada/jarvis_data_eng_NazaninAkbari/tree/feature/java-apps/core_java)]:
+      
+  -  Twitter App: Currently working on the project.
+  -  JDBC App: Currently working on the project.
+  - Grep App: Implemented Java Grep application which is looking for a regular expression that is searched in each file. I wrote this program in Java, using Streams, Lambda, and RegEx pattern matching, with Maven as the build tool. It is configured to be run as a command line with the JRE installed or via a docker container.
+
+
+## Highlighted Projects
+**Airline reservation system** [[GitHub](https://github.com/Na-zi/Projects/tree/main/airline-reservation)]: Designed an Airline Reservation System application which enables the customers to search and book the flights. This project has been developed in Visual Studio using C# programming language and Microsoft Azure SQL Database is implemented to manage the database on Cloud. The unit test and integration test are done for the system. Managing software development is done by using GitHub. Planning and tracing Scrum processes and activities are done by Microsoft Azure DevOps. Flow diagram and context diagram are used to model different sprints.
+
+**Snake game** [[GitHub](https://github.com/Na-zi/Projects/tree/main/snake_game)]: Designed snake game using Python as Object Oriented Language. The snake game has a snake that you can control it by using your keyboard to grab the food. The game is developed in PyCharm. I broke the problem down into different seven steps to solve it.
```

Comment:
**Snake game** [[GitHub](https://github.com/Na-zi/Projects/tree/main/snake_game)]: Designed snake game using Python as Object-Oriented Language. The snake game has a snake that you can control by using your keyboard to grab the food. The game was developed in PyCharm. I broke the problem down into seven different steps to solve it.

more tech details